build.gradle

```diff
@@ -122,6 +122,8 @@ repositories {
 
 dependencies {
     compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonVersion"
+    compile "com.auth0:java-jwt:3.3.0"
+    compile "org.apache.httpcomponents:httpclient:4.5.12"
```

Comment:
please remove the apache httpclient dependency, as we switched to the embedded [java.net.http.HttpClient](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpClient.html)

Reply:
the dependency is now removed